### ISSUE TITLE
Adding `noData` series is unnecessary?

### ISSUE THREAD

**Issue by shijistar** [ahmadsoe/ember-highcharts#93]
I'm new to `ember-highcharts`. I'm guessing whether the code snippet in `buildOptions` property of `high-charts` component is necessary or not?
```js
let chartContent = get(this, 'content.length') ? get(this, 'content') : [{
      id: 'noData',
      data: 0,
      color: '#aaaaaa'
    }];
```
When the `content` is empty, we will automatically add a `fake` series. However if we don't include the `no-data-to-display` module of `high-charts` library, the chart wouldn't show the `options.lang.noData` message anyhow, even we added a 'fake' series. On the other side, if we included the `no-data-to-display` module, we don't have to to the same thing neither because that module will automatically add one too.

Another big difference between the above solution and `no-data-display` module is that, I can't hide the x-axis and y-axis anyhow, even I set `options.plotOptions.yAxis/xAxis.showEmpty` to `false`, however if we remove the above code and use `no-data-display` module, the x-axis and y-axis will disappear and only a clear `noData` message is visible.

Is it possible to evaluate the two solutions? and do we have a solution to only show `noData` message without x-axis and y-axis?

**Comment by gregjopa** [comment]
I think we should support the `no-data-to-display` module. It seems like the standardized way of handling the "no data" use case. 

Including the `no-data-to-display` module should still be optional and we should fallback to something reasonable (perhaps the current solution).

Can you send a PR for this?

**Comment by shijistar** [comment]
I had also tried to show a label of noData message as a second fallback when highcharts 'no-data-to-display' module was not imported, just similar as what the module did. But I failed to align the label at center of the chart both in horizontal and vertical. Here is the sample code, you can continue working on it if you have interests.
```javascript
buildOptions: computed('chartOptions', 'content.[]', function() {
    let chartOptions = $.extend(true, {}, get(this, 'theme'), get(this, 'chartOptions'));
    let defaults = {};
    let chartContent = get(this, 'content');
    // if 'no-data-to-display' module has been imported, keep empty series and leave it to highcharts to show no data label.
    if (!get(this, 'content.length') && !Highcharts.Chart.prototype.showNoData) {
        let defaultOptions = Highcharts.getOptions();
        var hasLabels = (chartOptions.labels && chartOptions.labels.items && chartOptions.labels.items.length);
        // Preferred to show chart labels rather than an empty series. 
        if (defaultOptions.lang.noData && !hasLabels) {
            defaults = assign(defaults, {
                labels: {
                    items: [{
                        html: defaultOptions.lang.noData,
                        style: {
                            fontWeight: 'bold',
                            color: '#666',
                            // There's no way to place the label at the exactly center, so it's not good enough. I don't know if there is a way for that.
                            left: 400,
                            top: 200
                        }
                    }]
                }
            });
        } else {
            // This is the very original implementation, keep it as the last fallback. 
            // However don't know how it works for promoting no data message, at least the data should be an array?
            chartContent = [{
                id: 'noData',
                data: 0,
                color: '#aaaaaa'
            }];
            defaults = assign(defaults, { series: chartContent });
        }
    }

    return assign(defaults, chartOptions);
  }),
```

**Comment by muhammad-05** [comment]
hi all. I am also new to this topic. did the installation as the instructions. but with parameters and output graphs does not work. if there is a specialist ready to pay for the work that would chart displayed on the website.

**Comment by gregjopa** [comment]
Fixed by #98
